mlrun/alerts/alert.py

```diff
@@ -54,6 +54,62 @@ def __init__(
         created: str = None,
         count: int = None,
     ):
+        """
+        Alert config object
+
+        Example::
+            # create an alert on endpoint_id, which will be triggered to slack if there was an event of:
+            "data_drift_detected" 3 times in the next hour.
+            from mlrun.alerts.alert import AlertConfig
```

Comment:
General question - Can we also import from the parent module?
Meaning `mlrun.alerts.AlertConfig`?

Reply:
yes

Reply:
So let's change it to
```suggestion
            from mlrun.alerts import AlertConfig
```